__tests__/topics.test.js

```diff
@@ -0,0 +1,34 @@
+const db = require("../db/connection");
+const request = require("supertest");
+const testData = require("../db/data/test-data");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+
+afterAll(() => db.end());
+beforeEach(() => seed(testData));
+
+describe("GET /api/topics", () => {
+  it("returns an array of objects with description and slug", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        const topics = [
+          {
+            description: "The man, the Mitch, the legend",
+            slug: "mitch",
+          },
+          {
+            description: "Not dogs",
+            slug: "cats",
+          },
+          {
+            description: "what books are made of",
+            slug: "paper",
+          },
+        ];
+
+        expect(res.body).toEqual(topics);
```

Comment:
The test is good for this endpoint as it's simple but be careful of `toEqual` for GET endpoints as it requires it to be exactly the right response and for larger endpoints or ones that have additional data requirements then it may prove inflexible in the future. 

Don't need to change anything for this one, just something to be aware of 👍